test/Microsoft.IdentityModel.TestUtils/KeyingMaterial.cs

```diff
@@ -605,6 +609,23 @@ public static RsaSecurityKey RsaSecurityKey_2048
             }
         }
 
+        public static RsaSecurityKey RsaSecurityKeyCng_2048
+        {
+            get
+            {
+                if (_rsaCng == null)
```

Comment:
This looks like it could be simplified with a Lazy<T>

Reply:
I will make that change as i hardly ever use Lazy and perhaps this will make it top of mind.